api/v3/Job.php

```diff
@@ -370,6 +370,19 @@ function civicrm_api3_job_fetch_bounces($params) {
 }
 
 /**
+ * Metadata for bounce function.
+ *
+ * @param array $params
+ */
+function _civicrm_api3_job_fetch_bounces_spec(&$params) {
+  $params['is_create_activities'] = array(
```

Comment:
no you may not! but if you look at the line below this comment, that is why I'm sure the var will always be set. 

I'll fix the data block